crustcrawler:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Position Controllers ---------------------------------------
  joint_1_position_controller:
    #type: effort_controllers/JointEffortController
    #type: effort_controllers/JointPositionController
    type: position_controllers/JointPositionController
    joint: joint_1
    #actuator: joint_1_motor
    pid: {p: 100.0, i: 0.01, d: 10.0}
    #pid: {p: 1.0, i: 0, d: 0}
  joint_2_position_controller:
    #type: effort_controllers/JointEffortController
    #type: effort_controllers/JointPositionController
    type: position_controllers/JointPositionController
    joint: joint_2
    #actuator: joint_2_motor
    pid: {p: 100.0, i: 0.01, d: 10.0}
    #pid: {p: 1.0, i: 0, d: 0}
  joint_3_position_controller:
    #type: effort_controllers/JointEffortController
    #type: effort_controllers/JointPositionController
    type: position_controllers/JointPositionController
    joint: joint_3
    #actuator: joint_3_motor
    pid: {p: 100.0, i: 0.01, d: 10.0}
    #pid: {p: 1.0, i: 0, d: 0}
  joint_4_position_controller:
    #type: effort_controllers/JointEffortController
    #type: effort_controllers/JointPositionController
    type: position_controllers/JointPositionController
    joint: joint_4
    #actuator: joint_4_motor
    pid: {p: 100.0, i: 0.01, d: 10.0}
    #pid: {p: 1.0, i: 0, d: 0}
  joint_5_position_controller:
    #type: effort_controllers/JointEffortController
    #type: effort_controllers/JointPositionController
    type: position_controllers/JointPositionController
    joint: joint_5
    #actuator: joint_5_motor
    pid: {p: 100.0, i: 0.01, d: 10.0}
    #pid: {p: 1.0, i: 0, d: 0}
  joint_6_position_controller:
    #type: effort_controllers/JointEffortController
    #type: effort_controllers/JointPositionController
    type: position_controllers/JointPositionController
    joint: joint_6
    #actuator: joint_6_motor
    pid: {p: 100.0, i: 0.01, d: 10.0}
    #pid: {p: 1.0, i: 0, d: 0}
  joint_right_finger_position_controller:
    #type: effort_controllers/JointEffortController
    #type: effort_controllers/JointPositionController
    type: position_controllers/JointPositionController
    joint: right_finger_joint
    #actuator: righ_finger_joint_motor
    pid: {p: 100.0, i: 0.01, d: 10.0}
    #pid: {p: 1.0, i: 0, d: 0}
  joint_left_finger_position_controller:
    #type: effort_controllers/JointEffortController
    #type: effort_controllers/JointPositionController
    type: position_controllers/JointPositionController
    joint: left_finger_joint
    #actuator: left_finger_joint_motor
    pid: {p: 100.0, i: 0.01, d: 10.0}
    #pid: {p: 1.0, i: 0, d: 0}
